abcclassroom/dirsetup.py

```diff
@@ -0,0 +1,94 @@
+"""
+abc-classroom.dir-setup
+========================
+
+"""
+
+import os
+import shutil
+import argparse
+import abcclassroom
+
+
+def path_to_example(dataset):
+    """
+    Gets the path to an example dataset.
+
+    Parameters
+    ----------
+    dataset : string
+        stringy boi
```

Comment:
@nkorinek is this from @mbjoseph or you? 
it's kind of hilarious.

Reply:
That is awesome - but I'm not that funny. My money's on @nkorinek!

Reply:
@mbjoseph @nkorinek the mystery shall persist...

Reply:
Ooops, a test docstring made it through haha. My bad!

Reply:
😆 !!! it was you!! hilarious. thanks for the laugh. :) there is a funny meme account (that is no longer) that was for ultra running. and the called various shoes "boi"... so things like fat"bois" were the hoka super padded shoes, etc... hilarious.